xwiki-platform-core/xwiki-platform-flavor/xwiki-platform-flavor-api/src/main/resources/ApplicationResources_fr.properties

```diff
@@ -18,7 +18,7 @@
 # 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 # ---------------------------------------------------------------------------
 
-### Missing: flavor.picker.noflavor=No Flavor, let the wiki be empty.
+flavor.picker.noflavor=Aucune Saveur, garder le wiki vide.
```

Comment:
Not quite sure "Saveur" is the right word to use there... any idea? :)

Reply:
I think we said that we would keep "Flavor" in French.